crates/flipperzero/src/subghz/device.rs

```diff
@@ -0,0 +1,193 @@
+//! Safe wrapper utilizing the API provided in `lib/subghz/devices/devices.h`
+
+use super::{error::SubGhzError, preset::SubGhzPreset};
+use core::{ffi::CStr, ptr::null_mut};
+use flipperzero_sys::{
```

Comment:
I feel like importing every function from `sys` only to call it once complicates this source so I suggest aliasing `flipperzero_sys` as `sys` and using the qualified paths, i.e.:

```rust
use flipperzero_sys as sys;
```

and

```rust
let result = unsafe { sys::subghz_devices_begin(self.device) };
```